packages/react-ai/src/components/AIConversation/createAIConversation.tsx

```diff
@@ -16,13 +21,14 @@ export function createAIConversation<T extends Partial<AIConversationElements>>(
 ): {
   AIConversation: AIConversation<T>;
 } {
-  const { elements } = input;
+  const { elements, suggestedPrompts, actions } = input;
 
-  const Provider = createProvider({ elements });
+  const Provider = createProvider({ elements, actions, suggestedPrompts });
 
-  function AIConversation(): JSX.Element {
+  function AIConversation(props: AIConversationProps): JSX.Element {
+    const { messages, avatars } = props;
     return (
-      <Provider>
+      <Provider messages={messages} avatars={avatars}>
```

Comment:
Moved all these values that we previously mocked to be passed in from the example app

Reply:
What's the rationale behind passing `messages` and `avatars` as props versus `suggestedPrompts` and `actions` as input to `createAIConversation`?

Reply:
This is because suggestedPrompts and actions can be determined at build time and can be defined outside the scope of the React component.

Messages will be fetched async so must be passed in inside the react component and can't be done in `createAIConversation`.

Avatar is for a similar reason in that Avatar information can be updated based on react context or anything the developer decides at runtime, so should be changable with react props as well

Reply:
Got it